apollo-router-core/src/query_planner/selection.rs

```diff
@@ -42,23 +42,17 @@ pub(crate) struct InlineFragment {
     selections: Vec<Selection>,
 }
 
-pub(crate) fn select(
+#[cfg(test)]
+pub(crate) fn select<'a>(
```

Comment:
:screwdriver: Move this function to the tests module directly

:yarn: Check if you can update the tests and remove it entirely.

Reply:
:+1:

Reply:
done in 6a2573b